frontend/www/js/omegaup/components/problem/SearchBar.test.ts

```diff
@@ -6,19 +6,30 @@ import problem_SearchBar from './SearchBar.vue';
 
 describe('SearchBar.vue', () => {
   it('Should handle empty initial values', async () => {
+    const languages: { [key: string]: string } = {
+      all: T.wordsAll,
+      en: T.wordsEnglish,
+      es: T.wordsSpanish,
+      pt: T.wordsPortuguese,
+    };
     const wrapper = shallowMount(problem_SearchBar, {
       propsData: {
         columns: ['title', 'quality', 'difficulty'],
         initialColumn: '',
         initialMode: '',
         initialLanguage: '',
         initialKeyword: '',
-        languages: ['all', 'en', 'es', 'pt'],
+        languages: Object.keys(languages),
         modes: ['asc', 'desc'],
         tags: [],
       },
     });
 
     expect(wrapper.text()).toContain(T.wordsFilterByLanguage);
+
+    for (const language of Object.entries(languages)) {
+      const value = language[1];
+      expect(wrapper.find('select').text()).toContain(value);
```

Comment:
```suggestion
      expect(wrapper.find('select').text()).toContain(language);
```

por qué se necesita buscar sólo la segunda letra del lenguaje en vez de todo el lenguaje?

Reply:
Ah, porque `languages` contiene un diccionario de datos y con el `Object.entries(languages)` separo la clave del valor. Pero con este cambio ya debe quedar más claro:
```js
    for (const [_key, language] of Object.entries(languages)) {
      expect(wrapper.find('select').text()).toContain(language);
    }
```